bioptim/misc/optimal_control_program.py

```diff
@@ -20,7 +20,7 @@
 from ..dynamics.dynamics_type import DynamicsTypeList, DynamicsTypeOption
 from ..gui.plot import CustomPlot
 from ..interfaces.biorbd_interface import BiorbdInterface
-from ..interfaces.integrator import RK4
+from ..interfaces.integrator import RK4, IRK
```

Comment:
Is this an IRK4? Or the 4 would not make sense?

Reply:
No it is an IRK of order 'degree' (option for the user)